backend/app/api/endpoints/auth.py

```
@@ -53,3 +63,27 @@ def login(db: Session = Depends(get_db), login_data: LoginRequest = Body(...)):
     access_token = create_access_token(data={"sub": user.user_name, "user_id": user.id})
 
     return LoginResponse(access_token=access_token, token_type="bearer")
+
+
+@router.get(
+    "/admin-password/status",
+    response_model=AdminPasswordSetupStatusResponse,
+)
+def get_admin_password_setup_status(
+    db: Session = Depends(get_db),
```

Ruff (0.15.17)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)